Sources/JNI/JNIFields.swift

```diff
@@ -25,6 +25,14 @@ public extension JNI {
         try checkAndThrowOnJNIError()
         return try T.fromField(fieldID!, on: javaObject)
     }
+
+    func GetField(_ fieldName: String, fieldJavaClassName: String, from javaObject: JavaObject) throws -> JavaObject {
+        let env = self._env
+        let javaClass = try GetObjectClass(obj: javaObject)
+        let fieldID = env.pointee.pointee.GetFieldID(env, javaClass, fieldName, "L\(fieldJavaClassName);")
```

Comment:
I'm not sure but I think we might need to delete the local ref to the class here after we use it

Reply:
oh yes, totally oversaw this. we do it also in the other methods above :+1